extensions/blocks/recurring-payments/index.js

```diff
@@ -30,6 +30,7 @@ export const settings = {
 		_x( 'sell', 'block search term', 'jetpack' ),
 		_x( 'subscription', 'block search term', 'jetpack' ),
 		'stripe',
+		_x( 'membership', 'block search term', 'jetpack' ),
```

Comment:
Should it be `memberships`? I am not sure I understand when we should use the plural form, and when we should use singular? Is there a preferred way? Do we use the plural form in other places so we return results for folks pasting in the whole plural word instead of typing it in and getting the suggestion for the singular word?

Reply:
I think `memberships` makes more sense, since that way `membership` will also match.